src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,69 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
+
+    before do
+      User.current = user
+    end
+
+    context 'for users' do
```

Comment:
The whole test is for users, right? So this should be sth like ``for logged in users``?